firewall/app/tests/test_api.py

```diff
@@ -31,66 +31,19 @@ def create_app(self):
 	def test_app_is_production(self):
 		self.assertTrue(app.config['DEBUG'] is False)
 
-class TableTest(TestCase):
-	def create_app(self):
-		app.config.from_object('app.main.config.TestingConfig')
-		return app
-
-	def test_successful_create(self):
-		response = self.client.post('/table/', data=dict(family='inet', tablename='filter'))
-		self.assertEqual(201, response.status_code)
-
-	def test_successful_get(self):
-		response = self.client.get('/table/')
-		self.assertEqual(200, response.status_code)
-
-class ChainTest(TestCase):
-	def create_app(self):
-		app.config.from_object('app.main.config.TestingConfig')
-		return app
+class TestAddrBan(TestCase):
+	def test_ban_ipv4(self):
+		self.assertTrue(True)
 
-	def test_successful_create(self):
-		response = self.client.post('/table/', data=dict(family='inet', tablename='filter'))
-		self.assertEqual(201, response.status_code)
-		response = self.client.post('/chains/', data=dict(
-			family='inet',
-			tablename='filter',
-			chainname='input',
-			type='filter',
-			hook='input',
-			device='null',
-			priority='0',
-			policy='accept'
-		))
-		self.assertEqual(201, response.status_code)
+	def test_ban_mac(self):
+		self.assertTrue(True)
```

Comment:
you may upgrade this

Reply:
j'ai créé un fichier de test séparé où j'ai mis les tests du firewall_manager

aka test_ban_mac, test_ban_ipv4

les tests de bandwidth viendron quand on aura la bandwidth

j'ai pas push ce fichier parce que j'ai encore une erreur d'import de merde juste pour ce fichier

tout le reste devrait etre bon maintenant